osf/migrations/0085_populate_notification_subscriptions.py

```diff
@@ -0,0 +1,40 @@
+import logging
+
+from django.db import migrations
+
+from osf.models import NotificationSubscription, PreprintProvider
```

Comment:
Don't ever import app-defined models directly in a migration. They change over time, and because the database isn't necessarily in sync with the model during migration execution, direct imports can lead to newer model changes breaking old migrations.

Django builds a (somewhat limited -- no custom managers without specifying them and no mixed-in functionality) representation of what the models should look like, according to the database, at this point in the migration tree. They're available via the arguments passed into any function called via `RunPython` -- `apps` and `schema_editor`. So, you can do something like `NotificationSubscription = apps.get_model('osf', 'notificationsubsciption')` to get the models.

See docs about [historical models](https://docs.djangoproject.com/en/2.0/topics/migrations/#historical-models) and [`RunPython`](https://docs.djangoproject.com/en/2.0/ref/migration-operations/#runpython)